packages/grafana-data/src/field/displayProcessor.ts

```diff
@@ -67,6 +67,8 @@ export function getDisplayProcessor(options?: DisplayProcessorOptions): DisplayP
     if (!isBooleanUnit(unit)) {
       unit = 'bool';
     }
+  } else if (!unit && field.type === FieldType.string) {
+    unit = 'string';
```

Comment:
so to get the default behavior you have to set a unit, but we do not have a "no unit unit", so you would be forced to set it to short or something

Reply:
we do appear to have a "none" unit?

![1018_2022-03-01-16-21_chrome](https://user-images.githubusercontent.com/46142/156207091-5bd5703d-0715-41ec-a9fe-4e44f2514491.png)

Reply:
ah, that's true! forgot about that one!